plugins/node/opentelemetry-instrumentation-restify/.tav.yml

```diff
@@ -1,3 +1,8 @@
 restify:
-  versions: "4.3.4 || 5.2.0 || 6.4.0 || 7.7.0 || ^8.4.0"
-  commands: npm run test
+  - versions: "^10.0.0 || ^11.1.0"
+    node: ">=18"
```

Comment:
Should we still test versions 10 and 11 for older node versions? According to [restify package.json](https://github.com/restify/node-restify/blob/cbd16efa3be36e7888ecccc15ee28eaa8fa6c5ef/package.json#L93) it is supported:

```
...
  "engines": {
    "node": ">=10.0.0"
  },
...
```

Reply:
update in the bellow segment for `node: "<18"`